libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/SendActivity.cs

```diff
@@ -49,7 +49,16 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             }
 
             var activity = await Activity.BindToData(dc.Context, dc.GetState()).ConfigureAwait(false);
-            var response = await dc.Context.SendActivityAsync(activity, cancellationToken).ConfigureAwait(false);
+            ResourceResponse response = null;
+            if (activity.Type != "message" 
+                || activity.Text != null 
+                || (activity.Attachments != null && activity.Attachments.Count > 0) 
+                || activity.Speak != null 
+                || activity.SuggestedActions != null)
```

Comment:
>activity.SuggestedActions [](start = 19, length = 25)

(activity.SuggestedActions?.Any() == false)

Reply:
SuggestedActions is a partial class.

---
In reply to: [360173527](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360173527) [](ancestors = 360173527)